CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Project Changelog
 =================
 
+Release 1.2.2
```

Comment:
Since this is introducing a deprecation the version should be increased to 1.3.0.